src/schedules/domain/schedule.ts

```diff
@@ -26,6 +26,9 @@ export class Schedule {
   @ApiProperty({ type: ScheduleStatus })
   status: ScheduleStatus;
 
+  @ApiProperty({ type: String })
+  description: String;
```

Comment:
Como você definiu essa coluna como nullable no banco, aqui tem que ser opcional também:
```suggestion
  description?: String;
```

Reply:
Além disso o tipo tem que ser `string` e não a classe `String`